kvmagent/kvmagent/test/unittest_tools/prepare_env.sh

```
@@ -176,6 +176,6 @@ copy_kvm_virtualenv_to_venv3() {
     source /root/venv3/bin/activate
     cd /root/.zguest/zstack-utility/kvmagent/kvmagent/test/unittest_tools/unittest_pypi_source/
     pip install -r requirements/requirements1.txt -i file://`pwd`/pypi/simple
-    pip install -r requirements/requirements3.txt -i file://`pwd`/pypi/simple --extra-index-url http://mirrors.aliyun.com/pypi/simple 
+    pip install -r requirements/requirements3.txt -i file://`pwd`/pypi/simple --extra-index-url https://pypi.tuna.tsinghua.edu.cn/simple
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)